rdmo/questions/models/catalog.py

```diff
@@ -135,7 +130,7 @@ def copy(self, uri_prefix, key):
         catalog = copy_model(self, uri_prefix=uri_prefix, key=key, **kwargs)
 
         # copy m2m fields
-        catalog.sites.set(self.sites.all())
+        # catalog.sites.set(self.sites.all())
```

Comment:
This is WIP, right?

Reply:
with copy the `current_site` will be set on the `sites` and `editors` of the new catalog
```py
# copy m2m fields
# for sites and editors, set only current site
current_site = Site.objects.get_current()
catalog.sites.set([current_site])
catalog.editors.set([current_site])
```